.changeset/strong-numbers-cover.md

```diff
@@ -0,0 +1,9 @@
+---
+"@primer/react-brand": patch
+---
+
+Add escape-hatch for no title in SubdomainNavBar
+
+```jsx
+<SubdomainNavBar title="" />
```

Comment:
What about making title optional so we don't need to pass an empty string as a `prop`?

I see the main point to keep it `required` is accessibility reasons, but passing an empty string === not rendering the title or the link right?